apps/masterbots.ai/components/routes/profile/user-card.tsx

```
@@ -0,0 +1,199 @@
+      if (response.status === 401) {
+        toast.error(response.statusText)
+      }
+      setIsLoading(false)
+    },
+    async onFinish(message) {
+      setLastMessage(message.content)
+    },
+  })
+  
+  // Handle response in effect
+  useEffect(() => {
```

Biome

[error] This hook specifies more dependencies than necessary: lastMessage

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)